frontend/src/assets/styles/styles.scss

```diff
@@ -5,3 +5,7 @@
 
 // Components
 @import './components/form-elements.scss';
+
+@import '@bcgov-nr/nr-fsa-theme/design-tokens/type-family.scss';

```

Comment:
We don't need this one, the font design token already be called in the stylesheet

Reply:
Thanks, I'll update it